OpPipelineStageReadWriteShared.scala

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
I thinks this file here is by mistake? you meant to update this one - https://github.com/salesforce/TransmogrifAI/blob/master/features/src/main/scala/com/salesforce/op/stages/OpPipelineStageReadWriteShared.scala ?